examples/ui/scaling.rs

```diff
@@ -0,0 +1,142 @@
+use bevy::{prelude::*, utils::Duration};
+
+const SCALE_TIME: u64 = 400;
+
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, SystemLabel)]
+struct ApplyScaling;
+
+/// This example illustrates the various features of Bevy UI.
```

Comment:
This comment is just copied from the UI example - I think something like:
```suggestion
/// This example illustrates the UIScale resource from bevy_ui
```
might be more useful.

As an aside, we should make sure all of  the examples have a matching structure. In particular, I don't think the constants and label should be above `main`, but I'm not sure about that.

Reply:
We have a Example style-guide, but it doesn't directly mention the placement of constants and labels (except if you consider them `Example logic`)

https://github.com/bevyengine/bevy/blob/main/.github/contributing/example_style_guide.md?plain=1#L11-L14